pkg/cmd/serviceaccount/svcaccountcmdutil/errorutils/error_sdk.go

```diff
@@ -0,0 +1,34 @@
+package errorutils
```

Comment:
@wtrocki should we move this file to SDK, lik the other SDKs have?

Reply:
+1 on adding this to SDK. Thank you

Reply:
Raised PR https://github.com/redhat-developer/app-services-sdk-go/pull/555.